docusaurus/docs/internal/code-base-works/localization.md

```diff
@@ -0,0 +1,127 @@
+# Localization
+
+## Internationalization (i18n)
+
+Any code producing messages, labels, numbers, dates, times, and the like should use the `i18n` store and translation strings to generate and format them instead of hardcoding English or American-isms anywhere.   Messages and number formatting uses [ICU templating](https://formatjs.io/docs/intl-messageformat) for very powerful pluralization and customizing.
+
+The `assets/translations` dir stores a YAML file with translations for each supported language.
+  - English is automatically used as the "fallback" if a particular key is missing from a non-English language.
+  - If you don't have a native translation for a particular key, just leave that key out of the language
+  - Do not duplicate the English string into other languages.
+
+Translations should be the largest phrase that makes sense as a single key, rather than several separate things rendered in a fixed order.
```

Comment:
I think rather than making an example which nobody would like to write 😅  we could directly explain that it's possible interpolate enumerations and pass attributes. 

Note: This should be part of internal documentation as well, so it may be a repetition, although we expose just this one. Maybe just move that part of documentation to the external at this point?

Reply:
Not re-writing the internal documentation. Trying to move the more complex info to the internal side and improving extensions docs in a reasonable manner. Resolving